client/gutenberg/editor/utils.js

```diff
@@ -12,17 +12,33 @@ import debugFactory from 'debug';
 
 const debug = debugFactory( 'calypso:gutenberg' );
 
-export const overrideAPIPaths = siteSlug => {
-	// TODO: no API support for now. We'll also need to handle authorization here.
-	apiFetch.use( () => noop );
+export const applyMiddlewares = siteSlug => {
+	//make authenticated calls using the WordPress.com REST Proxy
+	//bypassing the apiFetch call that uses window.fetch
+	//first middleware in, last out
+	apiFetch.use( options => {
+		return proxy(
```

Comment:
Optional: I think naming this `wpcomProxyRequest` would make it slightly more readable.

Reply:
I should write 1000 times "do not copy and paste from the examples". Updated on fca7b6135a91b47113af39793ebf0812a14bb32b